test/test_utils.py

```diff
@@ -107,10 +107,7 @@ def test_get_ip(self):
         ip = get_local_ip_address()
         if ip == "No internet connection":
             return
-        try:  # check whether ip is valid
-            ipaddress.ip_address(ip)
-        except ValueError:
-            self.fail("Invalid IP address")
```

Comment:
`self.fail()` was the old unittest-based way of doing it. It's not necessary so I've removed it